db/data/20210616072230_add_empty_string_to_nil_description.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddEmptyStringToNilDescription < ActiveRecord::Migration[6.1]
+  def up
+    User.find_each do |user|
+      if user.description.nil?
+        user.description = '自己紹介文はありません。'
+        user.save!
```

Comment:
```suggestion
      user.update!(description: '') if user.description.nil?
```

空文字`''`を入れるようにお願いします〜

Reply:
@komagata 

> 空文字''を入れるようにお願いします〜

空文字だとバリデーションに通らないことがわかったので、`自己紹介文はありません。` に変更してもらいました。
https://github.com/fjordllc/bootcamp/issues/2798#issuecomment-863007186

Reply:
@Daichi831 @machida なるほど、すみません、そちらの文言のままで大丈夫です〜

Reply:
承知しました！
再度レビューをお願いします！

Reply:
@Daichi831 文言はそのままで大丈夫ですが、コードの方の修正はお願いいたします〜